### DISCUSSION TITLE
Issue template config admonishes you "we didn't recognize that category!", suggest minor link change.

### DISCUSSION THREAD

**Discussion by blakeNaccarato** [renovatebot/renovate#28661] [Suggest an Idea]
### Tell us more.

https://github.com/renovatebot/renovate/blob/1556cbbb41ce5018050dcb11c80aa43640e68381/.github/ISSUE_TEMPLATE/config.yml#L4

Changing that link to the following (add `choose` to the path) fixes the issue and removes the banner admonishing the user for an unrecognized category:

- Old link with the admonishment: https://github.com/renovatebot/renovate/discussions/new 
- Fixed link: https://github.com/renovatebot/renovate/discussions/new/choose

I could submit a one-liner typo PR for it if you agree. In the future, am I generally welcome to submit brief one-file "typo" PRs without making noise about it in Discussions ahead of time?

**Comment by HonkingGoose** [comment]
I'm fixing this with PR:

- #28703

_Personal opinion:_ for obvious and simple things like this example, I would rather get a PR, than having a Discussion plus PR. I'll let a maintainer respond _officially_ though. :wink: 

But for complicated stuff, or when it's not obvious what's the correct fix: create a Discussion first. :smile:

**Reply by rarkins** [reply]
Docs/template PRs are welcome directly whenever that's faster than describing it
